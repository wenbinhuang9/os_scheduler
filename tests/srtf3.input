10 10 40
101 50 1