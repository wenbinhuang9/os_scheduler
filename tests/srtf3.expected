10 50 0 40
101 51 0 1
0 21